content/millennium/dstu2/scheduling/slot.md

```diff
@@ -43,34 +51,42 @@ Search for Schedules that meet supplied query parameters:
 
     GET /Slot?:parameters
 
+_Implementation Notes_
+
+- Valid ids for the `schedule.actor` and `-location` search parameters will be determined by the client and provided 
+  when integrating your application with the client's production environment. See [overview](#overview) for details.
+  
 ### Authorization Types
 
 <%= authorization_types(practitioner: true, patient: true, system: true) %>
 
 ### Parameters
 
- Name             | Required?                                                        | Type           | Description
-------------------|------------------------------------------------------------------|----------------|------------------------------------------------------------------------------------
-`_id`             | This, or `slot-type`                                             | [`token`]      | The logical resource id associated with the resource.
-`slot-type`       | Yes when using `schedule.actor` and\or `-location`, or `id`      | [`token`]      | The type of appointments that can be booked into this slot. Example: `12345`
-`schedule.actor`  | This and `slot-type`, or `id`                                    | [`reference`]  | A single or comma separated list of Practitioner references. Example: `Practitioner/12345`
-`-location`       | This and `slot-type`, or `id`                                    | [`reference`]  | A single or comma separated list of Location references. Example: `12345`
-`date`            | Yes when using `slot-type`                                       | [`date`]       | The Slot date-time. Example: `2016`
-[`_count`]        | No                                                               | [`number`]     | The maximum number of results to be returned per page. Defaults to `50`.
+ Name             | Required?                  | Type           | Description
+------------------|----------------------------|----------------|------------------------------------------------------------------------------------
+`_id`             | Yes or `slot-type`         | [`token`]      | The logical resource id associated with the resource.
+`slot-type`       | Yes or `_id`               | [`token`]      | The type of appointments that can be booked into this slot. Example: `http://snomed.info/sct|394581000`
+`schedule.actor`  | Yes when using `slot-type` | [`reference`]  | A single or comma separated list of Practitioner references. Example: `Practitioner/2578010`
```

Comment:
This doesn't account for actor OR location being required

Reply:
Was leaving that to the notes below. However, called out more explicitly with this commit: https://github.com/cerner/fhir.cerner.com/pull/170/commits/c6084723c28ef0e7e42c0637fa67941ac8ac8b91